tests/rptest/tests/maintenance_test.py

```diff
@@ -69,7 +71,7 @@ def _enable_maintenance(self, node):
         assert status[
             "draining"] == False, f"Node {node.name} in maintenance mode"
 
-        self.admin.maintenance_start(node)
+        self.rpk.cluster_maintenance_enable(node)
```

Comment:
Nit: it would be nice to keep both versions of the test: direct API access, and RPK-driven access (or do it as a @parametrize on the test case)

Reply:
Done